kolena/workflow/__init__.py

```diff
@@ -87,4 +88,5 @@
     "TestRun",
     "test",
     "define_workflow",
+    "Audio",
```

Comment:
Should be ordered with the rest of the test sample types

Reply:
done